internal/status/statuses.go

```diff
@@ -178,11 +187,15 @@ func buildGatewayStatus(gateway *graph.Gateway) GatewayStatus {
 
 	gwConds := conditions.NewDefaultGatewayConditions()
 	if validListenerCount == 0 {
-		gwConds = append(gwConds, conditions.NewGatewayNotAcceptedListenersNotValid())
+		gwConds = append(gwConds, conditions.NewGatewayNotAcceptedListenersNotValid()...)
```

Comment:
are we intentionally missing setting Programmed conditions per listener in this PR?

Reply:
I didn't know that was a requirement to set the condition on the listeners